### PR TITLE
Fix missing icon in menu group

### PR DESCRIPTION
**Pull request by netounet**

### CODE REVIEW

**Comment by joshlopes** [comment]
:+1:  on this -- this was working!

File: Resources/views/Menu/sonata_menu.html.twig [hunk 1]
```diff
@@ -60,6 +60,7 @@
         <a href="#">
             {% set translation_domain = item.attribute('label_catalogue') %}
             {% set icon = item.attribute('icon')|default('') %}
+            {{ icon|raw }}
```
Comment (Soullivaneuh) [inline Resources/views/Menu/sonata_menu.html.twig]: According to commit https://github.com/sonata-project/SonataAdminBundle/commit/df42796fde5d850f04440fb344cac25caad25268#diff-7aad62c164c47d66f90ce813f6f91c49R62, I think this should be on `spanElement`.

cc @rande

**Comment by Soullivaneuh** [comment]
@netounet Is this fixes a specific issue?

Can you provide before/after result of that and, if possible, the link where this bug was introduce?

Thanks.

**Comment by netounet** [comment]
This bug was introduced by #2947 on line 62~64

Before : Icon set for admin group is not displayed as expected in menu

After : The icon is shown accordingly to original behaviour

**Comment by Soullivaneuh** [comment]
> This bug was introduced by #2947 on line 62~64

Yes, on `spanElement` so.

Can you please fix it? See: https://github.com/sonata-project/SonataAdminBundle/pull/3015#discussion_r31931064

And then, squash your commits. :+1:

**Comment by netounet** [comment]
@Soullivaneuh juste commit a fix to handle comment bug
I moved icon and label_catalog from item attributes to item extras

**Comment by Soullivaneuh** [comment]
Well, this fix modify `MenuBuilder.php`, why? cc @rande

**Comment by netounet** [comment]
because icon and label_catalogue are set in MenuBuilder as attributes instead of extras

**Comment by Soullivaneuh** [comment]
> because icon and label_catalogue are set in MenuBuilder as attributes instead of extras

Yes but why changing it?

Stop me if I'm wrong, but the original problem was just the missing `{{ icon|raw }}`.

I think @rande should handle this PR better than me.

**Comment by joshlopes** [comment]
Because the icon and translation are not attributes -- are more likely to be extras, that changes makes total sense. :+1:

**Comment by netounet** [comment]
@Soullivaneuh because the knp bundle merge item attributes in <li>
So you finally have invalid html attribute if you keep them in attributes (as you can see in @joshlopes comment earlier)
Moving this to extras (and that's the purpose of it) fix the problem and has more sense

**Comment by netounet** [comment]
@vihuvac the purpose of this PR is to fix missing icon AND remove unwanted generated html attribute
@rande if you have a minute to take a look at this small PR, it will be great :) I don't see how to argue more clearly . . .

**Comment by Soullivaneuh** [comment]
Just tested it on sandbox, works like a charm. Thanks.

**Comment by netounet** [comment]
thks

**Pull request merged**